msm-thermal driver fails to load with the following error message:

    <4>[    0.760658] msm_thermal: Wrong number of cpu

This happens because msm8210 only has two CPUs, but the device tree has
four CPU sensors in the qcom,msm-thermal node.

--- a/arch/arm/boot/dts/msm8610.dtsi	2014-09-25 23:49:52.000000000 -0300
+++ b/arch/arm/boot/dts/msm8610.dtsi	2015-11-22 09:35:53.276000000 -0200
@@ -916,8 +916,7 @@
 		qcom,core-control-mask = <0xe>;
 		qcom,hotplug-temp = <110>;
 		qcom,hotplug-temp-hysteresis = <20>;
-		qcom,cpu-sensors = "tsens_tz_sensor5", "tsens_tz_sensor5",
-				"tsens_tz_sensor5", "tsens_tz_sensor5";
+		qcom,cpu-sensors = "tsens_tz_sensor5", "tsens_tz_sensor5";
 		qcom,vdd-restriction-temp = <5>;
 		qcom,vdd-restriction-temp-hysteresis = <10>;
 		vdd-dig-supply = <&pm8110_s1_floor_corner>;
